modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/dtx/DistributedTransaction.java

```diff
@@ -42,6 +44,11 @@
 public class DistributedTransaction {
 
     /**
+     * Class logger
+     */
+    private static final Log LOGGER = LogFactory.getLog(DistributedTransaction.class);
```

Comment:
Better stick to a single naming convention for loggers ? lower case is the most common I see.

Reply:
I think this is the correct one since this is a `static final`.